scripts/macpython-download-cache-and-build-module-wheels.sh

```diff
@@ -86,6 +86,9 @@ if [[ -n ${ITKPYTHONPACKAGE_TAG} ]]; then
 fi
 
 # Run build scripts
+if [[ -d /Users/svc-dashboard/D/P/ITKPythonPackage ]]; then
```

Comment:
Why is this hard-coded path needed?  I don't have that path on my computer.

If needed for supporting a dedicated build machine, then a comment should be added giving the name of the machine and who this code is needed for.

Reply:
I thought this is a path inside virtual machine running the CI.

Reply:
This is the directory on the build system. The name of the path is historical.